packages/xo-server-audit/src/index.js

```diff
@@ -154,6 +185,42 @@ class AuditXoPlugin {
     }
     return records
   }
+
+  async _checkIntegrity({ oldest = NULL_ID, newest }) {
```

Comment:
Why these default values are not handled by `audit-core`?

Reply:
It's `XO` need that's why it's implemented in `xo-server-audit` instead.